src/plugins/dashboard/public/dashboard_container/panel_placement/place_clone_panel_strategy.ts

```diff
@@ -11,7 +11,7 @@ import { cloneDeep, forOwn } from 'lodash';
 import { PanelNotFoundError } from '@kbn/embeddable-plugin/public';
 
 import { DashboardPanelState } from '../../../common';
-import { GridData } from '../../../common/content_management';
+import type { GridData } from '../../../server/content_management';
```

Comment:
You should not import server code into public. Move this type to common if needed.

Reply:
Type imports are not subject to cross-boundary import rules. The `GridData` type is based on the `gridDataSchema` which imports `@kbn/config-schema`. The `@kbn/config-schema` package should only ever be in server code. See https://github.com/elastic/kibana/pull/189476 for more info. So this type was moved to server to avoid importing unwanted packages in common.